lib/bulk-apply-applications.js

```diff
@@ -190,13 +193,16 @@ function bulkApplyApplications (args, callback) {
       }, done);
     }
 
-    function retrieveParentsForUser (profiles, done) {
-      if (applications[0].status !== 'approved') return done(null, profiles);
```

Comment:
In the previous version, if the status of ticket is approved, we don't lookup parents.
Is there a reason? @DanielBrierton @rosalanghammer